knowledge/Shower.ttl

```diff
@@ -59,18 +59,16 @@
     :domainIncludes :Shower ;
     :rangeIncludes :GrabBar ;
     rdfs:label "hasInShowerGrabBar" ;
-    # ToDo comment anpassen
     rdfs:comment """
-        True if has grab bars in the bathroom, false if not and undefined if unknown.
+        Describes grab bar that you can use to get into the shower.
     """ .
```

Comment:
Is "describes" a good fit here? The property is of type boolean and acts more like a yes/no answer.